### ISSUE TITLE
Conflict installing redmine_xapian 3.0.2 for Redmine 5.0.2 in Debian

### ISSUE THREAD

**Issue by pgutierrezn2** [xelkano/redmine_xapian#131]
Hi,
TLDR;
I hit the error you mention in the documentation: “No Xapian search engine interface for Ruby installed"
I suspect the packages I installed from Debian repositories (1.4.18.) are old (backports repositories have the same versions) and the only thing I have to do is compilling the 1.4.21.* versions from source, right?

**Configuration**
- Bitnami VM with Debian GNU/Linux 11 (bullseye)
- Redmine 5.0.2
- redmine_xapian plugin v3.0.2
- Other plugins installed:  WYSIWYG Editor plugin v0.27.0, Wiki Extensions plugin v0.9.4

**Steps to reproduce**
Install redmine_xapian according to instructions: 
- download and unzip plugin to /redmine/plugins (remove version from folder name)
- sudo apt-get install xapian-omega ruby-xapian
- sudo apt-get install libxapian-dev poppler-utils \
    antiword unzip catdoc libwpd-tools libwps-tools gzip unrtf catdvi \
    djview djview3 uuid uuid-dev xz-utils libemail-outlook-message-perl
- sudo apt list --installed |grep xapian
	libxapian-dev/stable,now 1.4.18-3 amd64 [installed]
	libxapian30/stable,now 1.4.18-3 amd64 [installed,automatic]
	ruby-xapian/stable,now 1.4.18-1 amd64 [installed]
	xapian-omega/stable,now 1.4.18-2 amd64 [installed]
- bundle install																			
- RAILS_ENV=production bundle exec rake db:migrate									
- RAILS_ENV=production bundle exec rake redmine:plugins:migrate NAME=redmine_xapian 		
- restart redmine
- I see the message when trying to configure the plugin in the redmine GUI

**Comment by picman** [comment]
Yes, in case of Bitnami stack compiling from sources seems to be the only way.

**Comment by pgutierrezn2** [comment]
I did what you said, but hit the exact same error again:
`No Xapian search engine interface for Ruby installed => Full-text search won't be available. Install a ruby-xapian package or an alternative Xapian binding (https://xapian.org)`
I must be doing something wrong or missing a key step.

These are the steps I followed:
- remove old  dependencies from repository:
     sudo apt remove libxapian-dev
     sudo apt remove libxapian30
  After removing the above two ruby-xapian and xapian-omega disappeared too
- Compile from source xapian-core-1.4.21, xapian-omega-1.4.21 and  xapian-bindings-1.4.21 following [the instructions](https://xapian.org/docs/install.html)
   I needed the following two extra dependencies for the compilation to run smoothly:
     sudo apt-get install libpcre2-dev
     sudo apt-get install libpq-dev
- Uninstalled redmine_xapian 
     cd [redmine-install-dir]
     rake redmine:plugins:migrate NAME=redmine_xapian VERSION=0 RAILS_ENV=production
      rm -rf plugins/redmine_xapian
- Restart apache
